src/bin/lfortran.cpp

```diff
@@ -2060,6 +2060,7 @@ int main_app(int argc, char *argv[]) {
     app.add_flag("--static", static_link, "Create a static executable");
     app.add_flag("--shared", shared_link, "Create a shared executable");
     app.add_flag("--no-warnings", compiler_options.no_warnings, "Turn off all warnings");
+    app.add_flag("--disable-style", compiler_options.disable_style, "Turn off style suggestions");
```

Comment:
Isn't this a better option:
```suggestion
    app.add_flag("--no-style-warnings", compiler_options.disable_style, "Turn off style suggestions");
```
Later we can figure out some unified options, perhasp `-W`.